public/js/src/bridge/view_changes_modal.tag

```diff
@@ -0,0 +1,56 @@
+<view_changes_modal>
+    <div id="{ modalId }" name="{ modalId }" class="modal modal-fixed-footer">
+        <div class="modal-content">
+            <h4>Changes</h4>
+            <ul class="collection" if="{ changes.length }">
+                <li class="collection-item" each="{ change in changes }">
+                    <a class="secondary-content" onclick="{ parent.deleteChange }"><i class="small material-icons red-text">delete</i></a>
+                    <code>{ change.method} { change.url.replace('http://shipit.dev.services.dmtio.net', '') }</code>
```

Comment:
ACK! Need to replace this. The idea is to make the URL nicer when displaying to user. I need to use the config value.